test/acceptance/features/bindAppToService.feature

```diff
@@ -331,6 +331,195 @@ Feature: Bind an application to a service
         Then Error message is thrown
         And Service Binding "binding-request-empty-app" is not persistent in the cluster
 
+    @negative
+    Scenario: Cannot create Service Binding with name and label selector
+        Given CustomResourceDefinition backends.stable.example.com is available
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo-empty-app
+                annotations:
+                    service.binding/host: path={.spec.host}
+                    service.binding/username: path={.spec.username}
+            spec:
+                host: example.common
+                username: foo
+            """
+        When Invalid Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-with-name-label-selector
+            spec:
+                bindAsFiles: false
+                application:
+                    name: gen-missing-app
+                    group: apps
+                    version: v1
+                    resource: deployments
+                    labelSelector:
+                      matchLabels:
+                        name: backend-operator
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo-empty-app
+            """
```

Comment:
the scenario is missing assertion part, i.e. we should assert that error is thrown (specific one) when invalid service binding is applied.

Reply:
done